src/odemis/acq/stream/_static.py

```diff
@@ -466,6 +466,19 @@ def dis_bbtl(v):
 
         super(StaticARStream, self).__init__(name, list(self._pos.values()), *args, **kwargs)
 
+        # show the selected band filter wavelength in the emission field
+        try:
+            em_range = data[0].metadata[model.MD_OUT_WL]
+            if isinstance(em_range, str):
+                unit = None
+            else:
+                unit = "m"
+            self.emission = VigilantAttribute(em_range, unit=unit,
+                                              readonly=True)
+
+        except KeyError:
+            logging.warning("No emission wavelength for CL stream")
```

Comment:
CL stream -> AR stream ?

Reply:
Also, it should not be a warning, but just info. It's very common to not have this metadata, as many systems don't have a filter wheel.